duckduckgo/iOS
# BSK referrer integration

Task/Issue URL: https://app.asana.com/0/72649045549333/1200810841420631/f
Tech Design URL: https://app.asana.com/0/481882893211075/1202694361990652/f
CC: @bwaresiak 

**Description**:
This PR enabled referrer header trimming on iOS



**Steps to test this PR**:
1. Visit https://privacy-test-pages.glitch.me/privacy-protections/referrer-trimming/?run&testid=3p%20navigation&delay=1000
3. Expand the results
4. 1p navigation header test should be trimmed to the hostname "https://privacy-test-pages.glitch.me/" (this is actually the result of the 3p navigation but is a quirk of the test page)
1. Visit https://privacy-test-pages.glitch.me/privacy-protections/referrer-trimming/?run&testid=3p%20tracker%20navigation&delay=1000
3. Expand the results
4. 1p navigation header test should be trimmed to the hostname "https://privacy-test-pages.glitch.me/" (this is actually the result of the 3p tracker navigation but is a quirk of the test page)
5. Do some general smoke testing especially clicking links from one domain to another and ensure functionality seems correct.6. 

_Additional Test_
1. Using the devtools apply `target="_blank"` to one of the 3p manual test links.
2. Click the link. The results should be the same as the above tests.



**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
